File: math/verify-prime.py
class Solution:
    # @param A : integer
    # @return an integer
    def isPrime(self, A):
        if A <= 1:
            return 0; 
        else:
            for i in range(2,A):
                if(A % i == 0):
                    return 0;
            return 1;

File: math/verify-prime.cpp
int Solution::isPrime(int A) {
    if(A <= 1)
        return 0; 
    else
    {
        for(int i=2; i< A;i++)
        { 
            if(A % i == 0)
                return 0;
        }
        return 1; 
    }  
}
